staging/src/k8s.io/client-go/kubernetes/typed/core/v1/secret.go

```diff
@@ -157,7 +157,7 @@ func (c *secrets) Watch(opts metav1.ListOptions) watch.AggregatedWatchInterface
 			VersionedParams(&opts, scheme.ParameterCodec).
 			Timeout(timeout).
 			Watch()
-		if err != nil && opts.AllowPartialWatch {
+		if err != nil && opts.AllowPartialWatch && errors.IsForbidden(err) {
 			// watch error was not returned properly in error message. Skip when partial watch is allowed
 			klog.V(6).Infof("Watch error for partial watch %v. options [%+v]", err, opts)
```

Comment:
Is it the expected behavior? An error is logged as Info with a very low level of 6.